yt_dlp/extractor/rtvslo.py

```diff
@@ -164,3 +166,28 @@ def _real_extract(self, url):
             'series': meta.get('showName'),
             'series_id': meta.get('showId'),
         }
+
+
+class RTVSLOShowIE(InfoExtractor):
+    IE_NAME = 'rtvslo.si:show'
+    _VALID_URL = r'https?://(?:365|4d)\.rtvslo.si/oddaja/[^/?#&]+/(?P<id>\d+)'
+
+    _TESTS = [{
+        'url': 'https://365.rtvslo.si/oddaja/ekipa-bled/173250997',
+        'info_dict': {
+            '_type': 'playlist',
+            'id': '173250997',
+            'title': 'Ekipa Bled',
+        },
+        'playlist_count': 18,
+    }]
+
+    def _real_extract(self, url):
+        playlist_id = self._match_id(url)
+
+        webpage = self._download_webpage(url, playlist_id)
+        all_urls = orderedSet(re.findall(r'<a[^>]+href="(/arhiv/[^"]+)"', webpage))
+
+        return self.playlist_from_matches(
+            all_urls, playlist_id, self._html_extract_title(webpage, playlist_id),
+            getter=lambda x: urljoin('https://365.rtvslo.si', x), ie=RTVSLOIE)
```

Comment:
- `playlist_from_matches()` already uses `orderedSet`
- `self._html_extract_title(webpage, playlist_id)` isn't correct, don't pass a 2nd arg

```suggestion
        webpage = self._download_webpage(url, playlist_id)

        return self.playlist_from_matches(
            re.findall(r'<a [^>]*\bhref="(/arhiv/[^"]+)"', webpage),
            playlist_id, self._html_extract_title(webpage),
            getter=lambda x: urljoin('https://365.rtvslo.si', x), ie=RTVSLOIE)
```